app/src/main/java/net/opendasharchive/openarchive/services/webdav/WebDAVLoginActivity.kt

```diff
@@ -234,89 +223,73 @@ class WebDAVLoginActivity : AppCompatActivity() {
                 if (TextUtils.isEmpty(space.host)) return
                 try {
                     URL(space.host).toURI()
-                } catch (malformedURLException: MalformedURLException) {
-                    //not a valid URL
-                    return
-                } catch (malformedURLException: URISyntaxException) {
+                } catch (malformedURLException: Exception) {
+                    //not a valid URL or incorrect syntax error
                     return
                 }
-                val siteUrl = StringBuffer()
-                siteUrl.append(space.host)
-                if (!space.host.endsWith("/")) siteUrl.append("/")
-
                 val sardine: Sardine = OkHttpSardine()
                 sardine.setCredentials(space.username, space.password)
                 try {
                     try {
                         sardine.getQuota(space.host)
                         sardine.list(space.host + "/files/" + space.username + "/")
```

Comment:
What's this `Sardine` thing doing here? Seems to still use the newer method which requires a username.

Reply:
This is used to get the available storage space, but I dont see it being used anywhere. Removed it for now.